frontend/public/locales/en/home.json

```diff
@@ -6,18 +6,31 @@
     "description": "Our call centres and Service Canada Centres operate Monday to Friday, from 8:30 a.m. to 4:00 p.m., your local time. They are closed on statuary holidays.",
     "link": "/contact-us",
     "cards": {
+      "request-call": {
+        "title": "Request a Call",
+        "description": "Ask us to call you by using eServiceCanada",
+        "link-text": "Request A Call",
+        "href": "requesthref"
+      },
       "call-us": {
         "title": "Call us",
         "description": "You can reach us at either of the numbers below.",
-        "toll-free": "Toll free:",
-        "direct": "Direct Number:",
-        "toll-number": "1-800-000-000",
-        "direct-number": "1-800-000-000"
+        "toll-free-description":"The hours of operation are 8:30 am to 4:30 pm local time, Monday to Friday",
+        "outside-description":"The hours of operation are 8:40 am to 4:30 pm Eastern time, Monday to Friday",
+        "toll-free": "Canada and the United States Toll-free:",
+        "tty":"Canada and the United States TTY:",
+        "outside": "Outside Canada and the United States:",
+        "toll-number": "1-800-800-277-9914",
+        "tty-number": "1-800-255-4786",
+        "outside-number": "1-613-957-1954 (Call collect)",
+        "toll-number-href": "tel:18002554786",
+        "tty-number-href": "tel:18002554786",
+        "outside-number-href":"tel:16139571954"
       },
       "find-office": {
         "title": "Find an office",
         "description": "Find a Service Canada Centre near you using the link below.",
-        "link-text": "Find a Service Canada Centre",
+        "link-text": "Find a Service Canada Centre Near You",
```

Comment:
Should this be sentence-cased?

Reply:
I'm not sure whatever is in the figma mockups is correct. The canada.ca style guide is pretty clear on the use of sentence-casing everywhere.

Reply:
I remember having this discussion during one of our scrum ceremonies and it was agreed what we use figma as the source of truth (even when it is not correct). So I'll mark this as resolved and let business worry about it later.